docs: corrections to tr documentation (#2007)
trisyoungs/duq: web/docs/userguide/modules/correlation/tr/_index.md

The `TR` module calculates a full set of partial total distribution functions, $t_{ij}(r)$ for each unique pair of atom types $i$ and $j$ present in the target configuration(s).

> The `TR` module uses equations from Keen, [D. A. (2001)](https://journals.iucr.org/paper?S0021889800019993) and the numbered equations are referenced from there.

Reviewer: ```suggestion
The `TR` module uses equations from [Keen, D. A. (2001)](https://journals.iucr.org/paper?S0021889800019993) and the numbered equations are referenced from there.
```

For a configuration containing $N$ distinct atom types there are $\frac{1}{2}N(N+1)$ unique partial total distribution functions $t_{ij}(r)$, where $i$ and $j$ are the $i^{th}$ and $j^{th}$ atom type respectively. The standard formalism for a total distribution function between atom types $i$ and $j$ is

$$ T_{ij}(r) = \sum^n_{i,j}c_i b_i b_j t_{ij}(r)  (31)$$

> Where $t_{ij}(r)$ is calucluated via:

Reviewer: ```suggestion
Where $t_{ij}(r)$ is calculated via:
```

$$ t_{ij}(r) = 4 \pi r c_j \rho_0 g_{ij}  (33)$$

## Note

> It is reccomended to compare the partials to the total that the user scales them via eqution 31.

Reviewer: ```suggestion
It is recommended to compare the partials to the total that the user scales them via equation 31.
```
Can we clarify this a bit? I'm not quite sure what it's trying to tell me!

> The partial tr that is output from the module does not include the scaling before the summation is done. It is recommended for comparison of the partials to the total TR that the user scales them via equation 31. This includes multiplying the partial output from Dissolve by $c_i$, $b_i$ and $b_j$.

Reviewer: ```suggestion
The partial $t(r)$ that are output from the module do not include the scaling before the summation is done. It is recommended for comparison of the partials to the total $T(r)$ that the user scales them via equation 31. This includes multiplying the partial output from Dissolve by $c_i$, $b_i$ and $b_j$.
```
